### PR TITLE
doc: Azure IoT Hub lib: reducing the number of notes

### PR DESCRIPTION
**Pull request by peknis**
This PR reduces the number of consecutive
notes, hopefully making the doc a bit more
readable.

### CODE REVIEW

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 1]
```diff
@@ -113,25 +113,27 @@ To connect to Azure IoT Hub using DPS, complete the following steps:
...
-#. Set :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` to the security tag used while :ref:`azure_iot_hub_flash_certs`.
-   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option if multiple server certificates are provisioned.
+#. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option to the security tag used while :ref:`azure_iot_hub_flash_certs`.
+   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
 
 
 Application integration
 ***********************
 
+This section describes how to initialize the library, use the DSP service, and connect to Azure IoT Hub.
```
Comment (jtguggedal) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:131]:
```suggestion
This section describes how to initialize the library, use the DPS service, and connect to Azure IoT Hub.
```

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 2]
```diff
@@ -113,25 +113,27 @@ To connect to Azure IoT Hub using DPS, complete the following steps:
...
 
    Alternatively, you can set the registration ID at run time.
-#. Set the :kconfig:option:`CONFIG_AZURE_IOT_HUB_DPS_REG_ID` option to the registration ID.
+#. Set the :kconfig:option:`CONFIG_AZURE_IOT_HUB_DPS_REG_ID` Kconfig option to the registration ID.
 
    You can also set the registration ID at run time.
-#. Set :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` to the security tag used while :ref:`azure_iot_hub_flash_certs`.
-   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option if multiple server certificates are provisioned.
+#. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option to the security tag used while :ref:`azure_iot_hub_flash_certs`.
+   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
```
Comment (jtguggedal) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:125]:
```suggestion
   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SECONDARY_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
```

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 3]
```diff
@@ -91,16 +91,16 @@ To connect to Azure IoT Hub without using DPS, complete the following minimum re
...
 
    The device ID must match with the one used while creating the certificates.
    You can also set the device ID at run time by populating the ``device_id`` member of the :c:struct:`azure_iot_hub_config` structure passed to the :c:func:`azure_iot_hub_connect` function when connecting.
    If the ``device_id.size`` buffer size is zero, the compile-time option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID`` is used.
 #. Make sure that the device is already registered with your Azure IoT Hub, or follow the instructions in `Registering the device with Azure IoT Hub`_.
-#. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option to the security tag used in :ref:`azure_iot_hub_flash_certs`.
+#. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option to the security tag used in :ref:`azure_iot_hub_flash_certs`.
 
-   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option if multiple server certificates are provisioned.
+   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
```
Comment (jtguggedal) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:103]: There's a pre-existing typo here that is fixed in #10813 (https://github.com/nrfconnect/sdk-nrf/pull/10813/files#r1165247589), but might as well be done here.

```suggestion
   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SECONDARY_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
```

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 4]
```diff
@@ -91,16 +91,16 @@ To connect to Azure IoT Hub without using DPS, complete the following minimum re
 1. In the `Azure Portal`_, navigate to :guilabel:`IoT Hub` and select the desired IoT hub.
 #. In the overview page, locate and copy the ``Hostname`` and configure :kconfig:option:`CONFIG_AZURE_IOT_HUB_HOSTNAME` to this address.
 
-   The hostname can also be set at run time.
-#. Set the option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID` to the device ID.
+   You can also set the hostname at run time.
```
Comment (wiba-nordic) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:94]: I'm pretty sure that "hostname" in running text should be "host name". It's "host name" once on the page, and "hostname" a whole bunch of times.

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/networking/azure_iot_hub.rst (hunk 4 above)
Comment (peknis) [reply doc/nrf/libraries/networking/azure_iot_hub.rst:94]: Changed all of them.

**Review by jtguggedal — APPROVED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Pull request merged**